### DISCUSSION TITLE
Offline / downloadable release of the docs

### DISCUSSION THREAD

**Discussion by Kenneth-Mata** [helix-editor/helix#10147] [General]
I think a proper offline/download pdf (or textinfo) release of the docs could be practical, perhaps pandoc could be used to convert the .md files to an hyperlinked pdf

**Comment by the-mikedavis** [comment]
The book is built with [mdBook](https://github.com/rust-lang/mdBook) and the mdBook wiki links out to some third party backends that can emit pdf and epub: https://github.com/rust-lang/mdBook/wiki/Third-party-plugins#backends
